### PR TITLE
python-setproctitle: init at 1.1.9

### PR DESCRIPTION
**Pull request by exi**

### CODE REVIEW

**Comment by globin** [comment]
Please use "python-setproctitle: init at 1.1.9"

**Comment by exi** [comment]
@globin so close :D fixed

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -23405,6 +23405,23 @@ in modules // {
...
+    name = "python-setproctitle-${version}";
+    version = "1.1.9";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/setproctitle/setproctitle-${version}.tar.gz";
+      sha256 = "1mqadassxcm0m9r1l02m5vr4bbandn48xz8gifvxmb4wiz8i8d0w";
+    };
+
+    meta = {
+      description = "A fast and complete Python implementation of Markdown";
```
Comment (globin) [inline pkgs/top-level/python-packages.nix]: Copy paste?

Comment (exi) [inline pkgs/top-level/python-packages.nix]: fixed

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -23405,6 +23405,23 @@ in modules // {
...
+    name = "python-setproctitle-${version}";
+    version = "1.1.9";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/setproctitle/setproctitle-${version}.tar.gz";
+      sha256 = "1mqadassxcm0m9r1l02m5vr4bbandn48xz8gifvxmb4wiz8i8d0w";
+    };
+
+    meta = {
+      description = "setproctitle module allows a process to change its title (as displayed by system tools such as ps and top)";
```
Comment (globin) [inline pkgs/top-level/python-packages.nix]: The desciption should not contain the module name

**Comment by globin** [comment]
Sorry for nitpicking

**Comment by exi** [comment]
No problem. Better now?

**Comment by globin** [comment]
Thanks :+1:

**Pull request merged**